lib/auth0/api/v2/users.rb

```diff
@@ -185,12 +185,137 @@ def user_logs(user_id, options = {})
+          validate_roles_array roles
+          path = "#{users_path}/#{user_id}/roles"
+          delete(path, { roles: roles })
+        end
+
+        # Add one or more roles to a specific user.
+        # @see https://auth0.com/docs/api/management/v2#!/Users/post_user_roles
+        #
+        # @param user_id [string] The user_id of the roles to add.
+        # @param roles [array] An array of role names to add.
+        def add_roles(user_id, roles)
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          validate_roles_array roles
+          path = "#{users_path}/#{user_id}/roles"
+          post(path, { roles: roles })
+        end
+
+        # Get all Guardian enrollments for a specific user
+        # @see https://auth0.com/docs/api/management/v2#!/Users/get_enrollments
+        #
+        # @param user_id [string] The user_id of the enrollments to get.
+        #
+        # @return [json] Returns Guardian enrollments for the given user_id.
+        def get_enrollments(user_id)
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          get "#{users_path}/#{user_id}/enrollments"
+        end
+
+        # Get all permissions for a specific user.
+        # @see https://auth0.com/docs/api/management/v2#!/Users/get_permissions
+        #
+        # @param user_id [string] The user_id of the permissions to get.
+        #
+        # @return [json] Returns permissions for the given user_id.
+        def get_permissions(user_id)
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          get "#{users_path}/#{user_id}/permissions"
+        end
+
+        # Remove one or more permissions from a specific user.
+        # @see https://auth0.com/docs/api/management/v2#!/Users/delete_permissions
+        #
+        # @param user_id [string] The user_id of the permissions to remove.
+        # @param permissions [array] An array of Permissions to remove.
+        def remove_permissions(user_id, permissions)
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          permissions = validate_permissions_array permissions
+          delete("#{users_path}/#{user_id}/permissions", permissions)
+        end
+
+        # Add one or more permissions from a specific user.
+        # @see https://auth0.com/docs/api/management/v2#!/Users/post_permissions
+        #
+        # @param user_id [string] The user_id of the permissions to add.
+        # @param permissions [array] An array of Permissions to add.
+        def add_permissions(user_id, permissions)
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          permissions = validate_permissions_array permissions
+          post("#{users_path}/#{user_id}/permissions", permissions)
+        end
+
+        # Removes the current Guardian recovery code and generates and returns a new one.
```

Comment:
Should just be "remove" for consistency with other descriptions being mostly singular.